README.md

```diff
@@ -9,23 +9,55 @@ to lower the barrier to participating in release votes.
 
 This is very early days... once more, you need to be sure about the state of the release before voting. If in doubt, re-run commands yourself, or refer to the [wiki](https://cwiki.apache.org/confluence/display/ZIPKIN/Verifying+a+Source+Release).
 
-## Just run it!
+## Running
 
-This requires Docker to be installed on your machine, but nothing else. Main idea is you need to look at release VOTE thread and enter a few things:
+### Via `check.sh` (recommended)
+
+This requires Docker to be installed on your machine, but nothing else.
 
 ```bash
+git clone https://github.com/openzipkin-contrib/apache-release-verification.git
+cd apache-release-verification
 ./check.sh brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635
 ```
 
-If you want to do further manual verification after the script is done, set the `NO_CLEANUP=1` env variable, like so:
+This uses the Docker image built on Docker Hub. To build the image locally instead, set the `BUILD_IMAGE=1` env variable:
+
+```bash
+env BUILD_IMAGE=1 ./check.sh \
+  brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635
+```
+
+If you want to do further manual verification after the script is done, even if all check passed, set the `NO_CLEANUP=1` env variable, like so:
 
 ```bash
-env NO_CLEANUP=1 ./check.sh brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635
+env NO_CLEANUP=1 ./check.sh \
+  brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635
 ```
 
 At the end of the run, the script will output commands you can use to enter a Docker container with all the results of automated verification, and do your manual work.
 
-## The hard way
+### Without `git`
+
+If you really don't want to run `git`, you can get around it like this:
+
+```
+bash -c '. <(curl -sSL https://raw.githubusercontent.com/openzipkin-contrib/apache-release-verification/master/check.sh) brave-karaf 0.1.2 --gpg-key BB67A050 --git-hash 3cf4ac6577eb0d4775d20f24814e7a0852fa1635'
+```
+
+(In case you're wondering: this is not a standard `curl|sh` setup because the script is interactive, so it needs STDIN to be your interactive terminal)
+
+### Directly via Docker
```

Comment:
since this is so very easy, maybe title it `Directly via Docker (the easiest way)`

Reply:
Using `check.sh` is marked as `(recommended)`, which in turn invokes Docker.

Reply:
I guess the difference is whether you need to clone this repo to get the check.sh file or not. Were we planning on making the check.sh hosted somehow so people could curl bash it?

Reply:
> since this is so very easy, maybe title it `Directly via Docker (the easiest way)`

The main difference is that if you use Docker directly, and you want to inspect the state of the filesystem after the run, you need to know the `docker commit; docker run; docker rm` dance. This is pretty much the only thing `check.sh` does. So depending on how much you know Docker, and what you want to do, this might actually be harder than using `check.sh`.

I'm currently inclined to leave this as is: anyone who's comfortable with running Docker directly will be fine with the instructions as they are. Anyone who's not will benefit from `check.sh`.

> Were we planning on making the check.sh hosted somehow so people could curl bash it?

See the section directly above "Directly via Docker". It does exactly this.

Reply:
> See the section directly above "Directly via Docker". It does exactly this.

I should read more carefully. Right. So we can just use the GitHub hosting of files in the repo for this purpose.

Reply:
Yep, methinks for the time being that's good enough.